ui/plugins/pluginpanel.html

```diff
@@ -179,13 +179,14 @@
             </a>
           </span>
         </li>
-        <li class="small" ng-repeat="(label, value) in visibleFields">
+        <li class="small" ng-repeat="(label, value) in visibleFields"
+            ng-hide="label === 'Build' && job.build_system_type !== 'buildbot'">
```

Comment:
This hides the build name, not just the link.
The conditional needs to be on the ng-switch below, so that for the TC case, the name is still displayed, it just isn't a link.